crates/env/src/engine/off_chain/mod.rs

```diff
@@ -127,7 +127,7 @@ impl EnvInstance {
         Ok(())
     }
 
-    /// Resets the off-chain environment to unintialized state.
+    /// Resets the off-chain environment to uninitialized state.
```

Comment:
No idea why this was not detected. I created https://github.com/paritytech/ink/issues/738 for it.